### ISSUE TITLE
End-of-line comments break highlighting (ported from main repo)

### ISSUE THREAD

**Issue by AdamantConlanger** [def-gthill/lexurgy-app#1]
TL;DR: This minimal example renders with no syntax highlighting at all on the second line.
Minimal example:
```
Feature +voiced #
Feature +long
```

Details: The syntax highlighting seems to work as intended in almost all cases. Weirdly, though, comments seem to break it in some very specific edge cases. Specifically, a comment preceded by at least one space on a non-empty feature-declaring or symbol-declaring line causes the remainder of the document to evade all syntax highlighting. This doesn't happen if the comment isn't preceded by a space, nor if it occupies its own line, nor if it's placed at the end of a different type of declaration, name, or rule.

**Comment by def-gthill** [comment]
This should be fixed in 1d1408b.
